.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,34 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main, ft/2.0.0 ]
+  pull_request:
+    branches: [ main, ft/2.0.0 ]
```

```suggestion
    branches: [ main, v* ]
  pull_request:
    branches: [ main, v* ]
```
To match the tagging syntax used in https://github.com/ScottBrenner/cfn-lint-action/blob/90f970711aa1b19575dd9522d2340c5bca6df803/.github/workflows/release.yml#L4-L5

---

Agreed 👍 Thanks @ScottBrenner